packages/babel-plugin-jest-hoist/package.json

```diff
@@ -17,6 +17,6 @@
   },
   "scripts": {
     "prepublish": "npm test",
-    "test": "../../bin/jest.js"
+    "test": "node -e \"const spawn = require('child_process').spawn, path=require('path'); spawn('node', [path.resolve('../../bin/jest.js')], {stdio:'inherit'})\""
```

Comment:
would you mind reverting these again? I don't think we need this yet. +1 for the originality though.